client/src/pages/SurveyDetail/index.jsx

```diff
@@ -16,6 +16,9 @@ import {
 
 import Snackbar from '../../components/Snackbar';
 import { EmployeeCompletionTable } from '../../components/EmployeeTable';
+
+import SlackModal from '../../components/SlackModal/SlackModal';
+
 import formatDate from '../../utils/formatDate';
```

Comment:
put formatDate on line 24 instead. importing components first and then utils is the appropriate order